FreeRTOS/FreeRTOS
# Defender demo cleanup

Description
-----------
A few code quality cleanup in defender demo:

- Updated variables representing sizes from uint to size_t type.
- Updated variables representing boolean from uint to bool type.
- Updated task status list from hard coded size to dynamic size allocated based on number of running tasks.
- Some comment improvements.


Test Steps
-----------
Built and ran defender demo. The demo built and ran successfully.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
